markdown/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -1,339 +1,223 @@
-			<td>Run Theme Editor server</td>
-  </tr>
-	  <tr>
-    <td>-n, --no-cache</td>
-    <td>Turns off caching for API resource data per storefront page. The cache lasts for 5 minutes before automatically refreshing.</td>
-  </tr>
-	  <tr>
-			<td>--theme-editor-port [port]</td>
-			<td>Run the Theme Editor on a different port</td>
-  </tr>
-	  <tr>
-			<td>-h, --help</td>
-			<td>output usage information</td>
-  </tr>
-</table>
+<a id="stencil-help"></a>
 
+## `stencil help`
 
----
-
-<a href='#stencil_stencil-push' aria-hidden='true' class='block-anchor'  id='stencil_stencil-push'><i aria-hidden='true' class='linkify icon'></i></a>
+Displays help and returns all options available for the specified command.
 
-## Stencil Push Options
+**Usage:**
 
-To see Stencil CLI's additional options for the stencil push command (described above), enter the following on your command line:
+```shell
+stencil help [<COMMAND>]
+```
 
-`stencil help push`
+**Example:**
 
-Your terminal screen should read similar to the table below.
+```shell
+~ $ stencil help
+Usage: stencil [options] [command]
 
-<table>
-	<tr>
-    <th>Option</th>
-    <th>Definition</th>
-	</tr>
-  <tr>
-    <td>-V, --version</td>
-    <td>outputs the version number</td>
-  </tr>
-  <tr>
-    <td>--host [hostname]</td>
-    <td>specify the API host (default: https://api.bigcommerce.com)</td>
-  </tr>
-	  <tr>
-			<td>-f, --file [filename]</td>
-			<td>specify the filename of the bundle to upload</td>
-  </tr>
-	  <tr>
-			<td>-s, --save [filename]</td>
-			<td>specify the filename to save the bundle as</td>
-		</tr>
-		<tr>
-			<td>-a, --activate [variationname]</td>
-			<td>This will skip the prompts that normally come up asking if you would like to activate the theme and to specify a variation.
+Options:
+  -V, --version  output the version number
+  -h, --help     output usage information
+...
+```
 
-You can either specify the variation name after the flag or leave it blank to select the first variation (Light for Cornerstone).</td>
-		</tr>
-	  <tr>
-			<td>-h, --help</td>
-			<td>Output usage information.</td>
-		</tr>
-</table>
+---
 
-You can use the `-f` or `--filename` option in cases where you have already run `stencil bundle` to bundle your theme, but the resulting .zip file has not yet been uploaded to BigCommerce. Use the generated .zip file's **filename** as a parameter to identify the generated file in your theme directory. An example of the command is outlined below.
+<a id="stencil-init"></a>
 
-`stencil push -f Cornerstone-2.3.2.zip`
+## `stencil init`
 
-In this example, `Cornerstone-2.3.2.zip` is the name of the file that was generated after running `stencil bundle`.
+Creates a `.stencil` file used to configure the live preview when `stencil start` is ran. The configuration information can be specified using the optional switches; if the configuration information is not specified via options, a prompt for the information will be displayed. 

```

Comment:
Creates a `.stencil` file used to configure the live preview when `stencil start` is run. The configuration information can be specified using the optional switches; if the configuration information is not specified via options, a prompt for the information will be displayed.